test/lit/passes/dce_all-features.wast

```diff
@@ -1,3 +1,7 @@
+;; NOTE: This test was ported using port_test.py and could be cleaned up.
+
+;; RUN: foreach %s %t wasm-opt --dce --all-features -S -o - | filecheck %s
+
 (module
   (memory 10)
   (type $ii (func (param i32 i32)))
```

Comment:
Where is the output for this file? E.g. the first function `$call-me` has no `CHECK`s, nor the second.

Reply:
Not sure why the checks were missing for this file. I just regenerated all these tests now and it looks fine.